### DISCUSSION TITLE
Set default value for radio

### DISCUSSION THREAD

**Discussion by keczanm** [formio/formio.js#4783] [General]
Hi!

Can you help me how can I set a default value for a radio button group? For example, I have a question:

**Which one is your favorite fruit?**
- Apple
- Orange
- Banana

And I need the Apple to be checked by default.

Thank you very much! :)
Matthew

**Comment by wag110894** [comment]
Here you go!
 {
      "label": "Which one is your favorite fruit?",
      "optionsLabelPosition": "right",
      "inline": false,
      "tableView": false,
      "values": [
        {
          "label": "Apple ",
          "value": "apple",
          "shortcut": ""
        },
        {
          "label": "Orange",
          "value": "orange",
          "shortcut": ""
        },
        {
          "label": "Banana",
          "value": "banana",
          "shortcut": ""
        }
      ],
      "key": "whichOneIsYourFavoriteFruit",
      "type": "radio",
      "input": true,
      "defaultValue": "apple"
    },

**Reply by keczanm** [reply]
Thank you very much! 
I just realized there is a default value option in the Radio component / Data tab... facepalm. :D 
Thanks again,
Matthew
